__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const db = require("../db/connection")
+const request = require("supertest")
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+afterAll(() => {
+    return db.end();
+});
+
+beforeEach(() => {
+    return seed(data);
+});
+
+describe("/api/topics",()=>{
+    test("GET 200, responds with array of all topic objects which have slug and description properties", ()=> {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+            const { topics } = body 
+            expect(topics.length).toBe(3)
+            topics.forEach((topic) => {
+                expect.objectContaining({
+                    slug: expect.any(String),
+                    description: expect.any(String)
+                })
+            })
+        })
+    })
+
+    test("GET 404, Responds with 404 error and message of 'Route does not exist'", ()=> {
```

Comment:
It's great you've included this test! It should perhaps be separate from this endpoint, as all non-existent routes will return 404 regardless of /topics :)